example/src/app.tsx

```diff
@@ -33,6 +33,27 @@ LocaleConfig.locales['en'] = {
 };
 LocaleConfig.defaultLocale = 'en';
 
+LocaleConfig.locales['ko'] = {
```

Comment:
This is what you need to do in your app, not here

Reply:
It's a proposed solution, and Read Me seems in need of revision! Thank you for answer.

Reply:
It's a proposed solution, and README seems in need of revision! Thank you for answer. @Inbal-Tish

Reply:
Thanks. I just merged a PR for the readme change. Changing the app.tsx is affecting the demo so it can't get in